sass/compat/themes/buddyboss-theme/_woocommerce-form-fields-select2.scss

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------
+// SELECT2 FORM FIELDS
+// ------------------------------------
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } .select2-container .select2-dropdown {
+	max-width: none;
+	border: 1px solid var(--bb-content-border-color); // Needed to use `px` as units and theme variable to match theme styles
+	box-shadow: none;
+}
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } .select2-container .select2-dropdown--below {
+	border-top-left-radius: 0; // Needed to override theme specific `select2` styles
+	border-top-right-radius: 0; // Needed to override theme specific `select2` styles
+}
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } .select2-container .select2-dropdown--above {
+	border-bottom-left-radius: 0; // Needed to override theme specific `select2` styles
+	border-bottom-right-radius: 0; // Needed to override theme specific `select2` styles
```

Comment:
Not a big deal since the values are the same for left and right, but better to keep the same standard in the entire code base ;)

- Medium: Use variables `#{ $alignment-left }` and `#{ $alignment-right }` for properties and values so that styles for RTL languages are generated correctly.

```scss
border-top-#{ $alignment-left }-radius: 0;
border-top-#{ $alignment-right }-radius: 0;
```

This rule applies to all CSS properties and values that have `left` and `right`. The above is an example of properties that this rule applies, and below is an example of a CSS property value using this rule:

```scss
float: #{ $alignment-left };
```